README.adoc

```diff
@@ -14,15 +14,21 @@ containers for private storage and computation that can be opened, closed, and
 managed by smart contracts autonomously.
 
 Keeps are maintained by stakers, actors who run nodes and have skin in the
-game. When a new keep is opened, the requisite number of stakers are chosen
-via a BLS-based random beacon to maintain the keep.
+game, and collect fees for operating the network. When a new keep is opened,
+the requisite number of stakers are chosen via a
+link:https://blog.keep.network/whats-in-a-beacon-12c34b0bc078[BLS-based random beacon]
+to maintain the keep, using a process called
+link:https://en.wikipedia.org/wiki/Sortition[sortition].
 
 The first type of keep launching with the network is the `BondedECDSAKeep`,
 allowing smart contracts to generate private keys and sign messages without
 endangering key material. ECDSA keeps mean decentralized signing, cross-chain
 applications, and new tools for custodial applications — from Solidity. This
 capability is used heavily by https://tbtc.network/[tBTC].
 
+To learn more about ECDSA keeps, check out
+[keep-ecdsa](https://github.com/keep-network/keep-ecdsa)
```

Comment:
Roses are red
Violets are blue
Periods are nice
And so is using them to end sentences 😘

Reply:
(Also this is asciidoc, not markdown. `https://github.com/keep-network/keep-ecdsa[keep-ecdsa]`)